packages/fuchsia_ctl/bin/main.dart

```diff
@@ -264,7 +268,8 @@ Future<OperationResult> test(
       identityFilePath: identityFile,
       command: <String>[
         'run',
-        'fuchsia-pkg://fuchsia.com/$target#meta/$target.cmx'
+        'fuchsia-pkg://fuchsia.com/$target#meta/$target.cmx',
+        arguments
```

Comment:
Are we sure this works properly with multiple arguments?  Do we nnot need something like `...arguments.split(' ')`?

Reply:
It works locally at least:

`./fuchsia_ctl -d yard-clone-baton-snare test -f shell_tests-0.far -t shell_tests -a "--gtest_filter=-ShellTest.HandlesActualIphoneXsInputEvents --gtest_also_run_disabled_tests"`

That correctly skips `ShellTest.HandlesActualIphoneXsInputEvents` and runs `ShellTest.DISABLED_ReportTimingsIsCalled`

Reply:
Works for me - LGTM